firewall/outgoing_blocker_default.go

```diff
@@ -27,35 +27,35 @@ const (
 )
 
 // DefaultTrackingBlocker traffic blocker bootstrapped for global calls
-var DefaultTrackingBlocker TrafficBlocker = &noopTrafficBlocker{}
+var DefaultTrackingBlocker OutgoingTrafficBlocker = &outgoingBlockerNoop{}
 
-// TrafficBlocker interface neededs to be satisfied by any implementations which provide firewall capabilities, like iptables
-type TrafficBlocker interface {
+// OutgoingTrafficBlocker interface neededs to be satisfied by any implementations which provide firewall capabilities, like iptables
```

Comment:
For future reference, godoc should be a complete sentence, notably ending with a dot.